projects/configs/tag-operations/test-extract-uclh-omop-cdm.yaml

```diff
@@ -491,6 +591,10 @@
   group: 0x0038
   element: 0x0500
   op: "delete"
+- name: "UID"
```

Comment:
```suggestion
- name: "UID Attribute"
```

Reply:
I don't understand this suggestion. Do we want to add `Attribute` to `[UID](https://dicom.innolitics.com/ciods/mr-image/specimen/00400560/00400620/0040a124)` tag? I just removed Attribute words in config files [bc0a4c8](https://github.com/SAFEHR-data/PIXL/pull/493/commits/bc0a4c865bd9fcacdddf15eec1399ded20d3431c) otherwise we need to add `Attribute` to all tags

Reply:
The human readable name from what I can see should be UID Attribute, not just UID